src/pages/Landing.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { useOidc } from '@axa-fr/react-oidc';
+import { TableCellsIcon, UserIcon } from '@heroicons/react/24/solid';
+import { getLogger } from '../logger';
+import { Panel, PanelBody } from '../components/Panel';
+import { Button } from '../components/Button';
+import './Landing.css';
+
+const logger = getLogger('<Landing>');
+
+const Landing = () => {
+  const { login, isAuthenticated } = useOidc();
+
+  return (
+    <Panel
+      fill={false}
+      className="landing-panel"
+    >
+      <PanelBody>
+        <h3 style={{ fontWeight: '600' }}>Welcome to the Philanthropy Data Commons pilot.</h3>
+        <p>
+          This data viewer displays the data in the Philanythropy Data Commons pilot.
```

Comment:
I picked this line to ask this question: do we care about internationalization at this stage of the project?  I assume the answer is no; sorry if I'm rehashing conversations that have already been had!

Reply:
That's a great question, and yes, we do care!

Unfortunately while we're accruing debt here, that ship has sailed (in that the site is going to be in user's hands in a matter of minutes without i18n) so that shouldn't block this particular PR. But it's important and I'm glad you noted it.

Reply:
`s/Philanythropy/Philanthropy/g`

Reply:
Oh nooooo